src/Instance/InstanceDetails/InstanceDetails.js

```diff
@@ -119,11 +120,14 @@ const InstanceDetails = ({
         <TitleManager record={instance.title} />
 
         <AccordionStatus>
-          <Row end="xs">
-            <Col
-              data-test-expand-all
-              xs
-            >
+          <Row>
+            <Col xs={10}>
+              {
+                instance.discoverySuppress &&
+                <TextField fullWidth readOnly value={intl.formatMessage({ id: 'ui-inventory.instance.suppressedFromDiscovery' })} error=" " />
```

Comment:
Wouldn't it be better to put these attributes on multiple lines for better readability?

Reply:
Looks good to me! Thanks, @mkuklis  for making `WarningMessage` more reusable.